### DISCUSSION TITLE
How does CPU time calculated in deno deploy?

### DISCUSSION THREAD

**Discussion by vicary** [denoland/deno#15988] [Q&A] [answered]
From the pricing page, CPU time is explicitly noted as "not wall clock time", but I have yet to find a documentation with such definition.

How does a it counts towards CPU time when, for example, SSR fetches a database?

**Comment by zachauten** [comment, marked as answer]
I think [Wikipedia](https://en.wikipedia.org/wiki/CPU_time) provides a sufficient definition. It means that time spent waiting on a network request would not be billed.
> CPU time (or process time) is the amount of time for which a central processing unit (CPU) was used for processing instructions of a computer program or operating system...
...
> In contrast, elapsed real time (or simply real time, or [wall-clock time](https://en.wikipedia.org/wiki/Wall-clock_time)) is the time taken from the start of a computer program until the end as measured by an ordinary clock.

**Reply by zachauten** [reply]
Also, I saw it mentioned in another thread that questions about Deno Deploy should go here: https://github.com/denoland/deploy_feedback

**Reply by vicary** [reply]
@zachauten Thanks for sharing the Wikipedia, I'll mark yours as answer before an official comment is made.

Discussion is not enabled at https://github.com/denoland/deploy_feedback. Since this is a general question rather than an actual request for a documentation change, hence, no actionable items, so I think it stays here.

**Reply by kitsonk** [reply]
[deploy_feedback](https://github.com/denoland/deploy_feedback) is the appropriate place for this, irrespective of discussions being enabled or not.

**Reply by vicary** [reply]
@kitsonk Should I open an issue there instead?

**Reply by kitsonk** [reply]
Yes.

**Reply by vicary** [reply]
Done, thanks for the heads up!
